app/templates/carpools/add_rider.html

```diff
@@ -20,7 +20,7 @@
             {{ form.hidden_tag() }}
             <h4>Join this carpool</h4>
             <h1>Confirm your details</h1>
-            <p>To join this carpool, please verify your profile information and make any changes as needed.</p>
+            <p>When you request a seat, your name, gender, and contact information from your profile will be shared with the driver.  You can update this information on <a href="/profile" target="_blank">your profile page</a>.</p>
```

Comment:
Please use `{{ url_for('auth.profile') }}` instead of the direct link to `/profile`.

Reply:
💯